_posts/en/newsletters/2019-04-02-newsletter.md

```diff
@@ -124,11 +124,11 @@ Now we can simply attempt to decode each of these addresses
 (0, [110, 175, 166, 4, 165, 3, 160, 187, 68, 90, 209,
      246, 218, 168, 15, 22, 43, 86, 5, 214])
 
-In [16]: segwit_addr.decode(HRP, typo_address)
-Out[16]: (None, None)
+>>> segwit_addr.decode(HRP, typo_address)
```

Comment:
Do you also want to add these `>>>` prompts to commands above?

Reply:
I do.  Done (force pushed).  Thanks!